polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,371 @@
+
+**Polkadot 1.0** currently achieves resilience through several strategies:
+
+1. **Nominated Proof of Staking (NPoS)**: This ensures that the stake per validator is maximized and evenly distributed among validators.
+
+2. **1KV Program**: Designed to encourage new operators to join the network, this program enhances both physical decentralization (the number of validator nodes per service provider) and social decentralization (the number of validator nodes per operator). Feel free to explore these metrics using the [Polkawatch App](https://polkadot.polkawatch.app/){target=\_blank}.
+
+3. **On-Chain Treasury and Governance**: Known as [OpenGov](link to opengov overview page){target=\_blank}, this system allows every decision to be made through public referenda, enabling any token holder to cast a vote.
+
+### Polkadot's Blockspace
+
+Polkadot 1.0’s design allows for the commoditization of blockspace.
+
+Blockspace is the capacity of a blockchain to finalize and commit operations, encompassing its security, computing, and storage capabilities. Its characteristics can vary across different blockchains, affecting **security**, **flexibility**, and **availability**.
+
+- **Security** measures the robustness of blockspace in Proof-of-Stake (PoS) networks, linked to the stake locked on validator nodes, the variance in stake among validators, and the total number of validators. It also considers social centralization (how many validators are owned by single operators) and physical centralization (how many validators run on the same service provider).
+
+- **Flexibility** reflects the functionalities and types of data that can be stored, with high-quality data essential to avoid bottlenecks in critical processes.
+
+- **Availability** indicates how easily users can access blockspace. It should be easily accessible, allowing diverse business models to thrive, ideally regulated by a marketplace based on demand and supplemented by options for "second-hand" blockspace.
+
+Polkadot is built on core blockspace principles, but there’s room for improvement. Currently, tasks like balance transfers, staking, and governance are managed on the relay chain. 
+
+Delegating these responsibilities to [system chains](add link to System Chains section) could enhance flexibility and allow the relay chain to concentrate on providing shared security and interoperability.
+
+Currently, blockspace is accessible only through auctions, and the auction winner gains access to a "freighter of blocks," whether needed or not. This setup creates high entry barriers and can lead to wasted energy and resources.
+
+!!!note
+    For more information about blockspace see this [interview](https://www.youtube.com/watch?v=e1vISppPwe4){target=\_blank} with **Robert Habermeier** as well as this [article](https://www.rob.tech/blog/polkadot-blockspace-over-blockchains/){target=\_blank} by him.
+
+## Protocol Componets
```

Comment:
I would entirely remove the content of the Protocol Components section, we can leave the titles of each component and link them to their specific pages. We have pages for accounts, cryptography, runtime upgrades, extrinsics,etc. If some of the content here was not explained in the existing pages it can be added there

Reply:
yes, perfect. I agree

Reply:
@themacexpert looks like this needs to be removed still

Reply:
Took this stuff out. We do handle all of those things on their own pages in the updated IA